server/resp/src/main/java/org/infinispan/server/resp/configuration/RespServerConfiguration.java

```diff
@@ -20,7 +20,8 @@
 public class RespServerConfiguration extends ProtocolServerConfiguration<RespServerConfiguration> {
 
    public static final int DEFAULT_RESP_PORT = 6379;
-   public static final String DEFAULT_RESP_CACHE = "respCache";
+   public static final String DEFAULT_RESP_CACHE_TEMPLATE = "org.infinispan.template.RESP";
+   public static final String DEFAULT_RESP_CACHE = "defaultRespCache";
```

Comment:
I don't think this should be changed as it is an API change.

Reply:
ok